packages/kbn-dev-utils/src/proc_runner/with_proc_runner.test.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ToolingLog } from '../tooling_log';
+import { withProcRunner } from './with_proc_runner';
+import { ProcRunner } from './proc_runner';
+
+it('passes proc runner to a function', async () => {
+  await withProcRunner(new ToolingLog(), async proc => {
```

Comment:
__optional nit:__ `async proc` --> `proc` here and tests below (`await` inside of `withProcRunner` should work fine even if callback doesn't return promise). But it's not a big deal, hence `optional`.